grails-app/views/admin/_permissionTablePaginated.gsp

```diff
@@ -1,4 +1,9 @@
-
+<form id="searchUser" class="hub-form-search-box1">
+    <div class="form-group row required">
+        <label for="email" class="control-label col-sm-2" style="left: -14px">Enter User's Email Address</label>
```

Comment:
Change label to "Search for email address".
Add a button with a search icon to press that will trigger the search.